app/assets/javascripts/govuk_publishing_components/modules.js

```diff
@@ -38,8 +38,13 @@
           if (typeof GOVUK.Modules[moduleName] === 'function' && !started) {
             // Vanilla JavaScript GOV.UK Modules and GOV.UK Frontend Modules
             if (GOVUK.Modules[moduleName].prototype.init) {
-              new GOVUK.Modules[moduleName](element).init()
-              element.setAttribute('data-' + moduleNames[j] + '-module-started', true)
+              try {
+                new GOVUK.Modules[moduleName](element).init()
+                element.setAttribute('data-' + moduleNames[j] + '-module-started', true)
+              } catch (e) {
+                // if there's a problem with the module, don't start it
```

Comment:
I'm wondering if this should this be more specific e.g.`if there's a problem with the module, catch its error so that other modules on the page will still start`

Reply:
Updated, thanks. I've also added a changelog entry as I think we should keep track of when this change was made.